common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -227,9 +239,127 @@ public boolean onTouch(final View view, final MotionEvent event) {
         mWebView.getSettings().setBuiltInZoomControls(webViewZoomControlsEnabled);
         mWebView.getSettings().setSupportZoom(webViewZoomEnabled);
         mWebView.setVisibility(View.INVISIBLE);
+        mWebView.getSettings().setPluginState(WebSettings.PluginState.ON);
```

Comment:
This one is deprecated right?
https://developer.android.com/reference/android/webkit/WebSettings.html#setPluginState%28android.webkit.WebSettings.PluginState%29